src/Symfony/Component/Config/Tests/Definition/ExpressionNodeTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Tests\Definition;
+
+use Symfony\Component\Config\Definition\ExpressionNode;
+use Symfony\Component\ExpressionLanguage\Expression;
+
+class ExpressionNodeTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        if (!class_exists(Expression::class)) {
+            $this->markTestSkipped('The ExpressionLanguage component is not installed.');
```

Comment:
no need for this one: EL is always here, thx to composer require-dev
BUT you need to fix the lowest required EL version in Config's composer.json (to fix travis)